tests/PHPUnit/Integration/SegmentTest.php

```diff
@@ -524,7 +524,7 @@ public function test_getSelectQuery_whenJoinConversionAndActionOnVisit_andPageUr
                 FROM
                     " . Common::prefixTable('log_visit') . " AS log_visit
                     LEFT JOIN " . Common::prefixTable('log_link_visit_action') . " AS log_link_visit_action ON log_link_visit_action.idvisit = log_visit.idvisit
-                    LEFT JOIN " . Common::prefixTable('log_conversion') . " AS log_conversion ON log_conversion.idvisit = log_link_visit_action.idvisit
+                    LEFT JOIN " . Common::prefixTable('log_conversion') . " AS log_conversion ON log_conversion.idvisit = log_visit.idvisit
```

Comment:
Here the order is not different but it uses the table from `FROM`. This should not  change anything and also not affect performance